products/sle/main.pm

```diff
@@ -887,7 +887,9 @@ sub load_x11tests {
     loadtest "x11/xterm";
     loadtest "x11/sshxterm";
     if (gnomestep_is_applicable()) {
-        loadtest "update/updates_packagekit_gpk" unless is_staging;
+        if (!sle_version_at_least('15')) {
```

Comment:
```
if(!sle_version_at_least('15') and !is_staging()){
  loadtest "update/updates_packagekit_gpk"     # bsc#1061243
}

```
would be more readable. Also it is not really clear what "# bsc#1061243" comment suppose to mean ? This test is for checking this bug ?  Could you add few word how this bug is related to this test ?